run_interactive_multiple_host.py

```diff
@@ -99,6 +97,7 @@ def main(argv):
     while True:
       # pylint: disable-next=all
       decode_state, result_tokens = engine.generate(None, decode_state)
+      result_tokens = result_tokens.convert_to_numpy()
```

Comment:
Why do we need to convert to numpy here?

Reply:
Follow the jetstream logic, convert to numpy and process later. https://github.com/google/JetStream/blob/e19a7906d8cdf1cae658a4c7c4f6f516aade49f9/jetstream/core/orchestrator.py#L708